pkg/raft/rafttest/doc.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2015 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package rafttest provides functional tests for etcd's raft implementation.
```

Comment:
`cockroachdb's raft implementation` 😃 

Worth a final grep for `"etcd"`?

Reply:
Made a few edits. There are still some etcd specifics, we would be removing them later as part of a larger clean-up aiming to retain only bits used by CRDB.